### DISCUSSION TITLE
Default MQTT server usability

### DISCUSSION THREAD

**Discussion by xhci** [meshtastic/Discussions#106] [General]
Hello all!

I'm trying to bridge two mesh networks via MQTT, and am confused by the default MQTT server configuration.

It's my understanding that it sets a 0-hop limit on all outgoing messages so that they aren't rebroadcast, however this creates a problem where a client can be heard and not receive a reply.

For example, I have a mobile client that broadcasts to a station node. That station node then relays the message to other nodes with MQTT enabled. One of those nodes replies, however the original receiving station node will never relay the reply back to the mobile client.

I understand that MQTT has the capability to clog up the mesh, but wouldn't a 1-hop limit make a little more sense for the public server? The station node in question is connected via satellite, and there's no cell service around for the mobile client to use. Maybe it's encouraged to use a private server for this kind of use-case?

Thanks!

**Comment by garthvh** [comment]
The public server is way too busy for any Lora mesh, 50 million packets a week

**Reply by xhci** [reply]
I agree that the main topics are too busy, but I think allowing some low number of hops for small topics would help geographically challenging regions bootstrap their meshes together.
